recipes/libressl/2.9.2/test_package/conanfile.py

```diff
@@ -0,0 +1,38 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class PackageTest(ConanFile):
+    settings = "os", "compiler", "arch", "build_type"
+    generators = "cmake"
+
+    def configure(self):
+        if tools.cross_building(self.settings):
+            del self.settings.compiler.libcxx
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.verbose = True
+        cmake.configure()
+        cmake.build()
+
+    def imports(self):
+        self.copy("*.so*", dst="bin", src="lib")
```

Comment:
I don't think imports are needed, use `run_environment=True`

Reply:
I didn't know this existed 😮